backend/src/main/java/pl/edu/agh/api/controller/ActorsController.java

```diff
@@ -14,13 +15,14 @@ public class ActorsController {
     private FighterQueryService fighterQueryService;
 
     @GetMapping("/fighters")
+    @ResponseBody
```

Comment:
I would consider using ResponseEntity instead of @ResponseBody annotation, it gives more flexibility, as you can set some headers.